applications/callflow/src/module/cf_log_in_out_ring_group.erl

```diff
@@ -0,0 +1,91 @@
+%%%-------------------------------------------------------------------
+%%% @doc
+%%% Logs a user in or out of a ring group
+%%% @end
+%%% @contributors
+%%%   Max Lay
+%%%-------------------------------------------------------------------
+-module(cf_log_in_out_ring_group).
```

Comment:
Also, I don't see the generated callflow schema in the commits. Please add it, along with filling out the descriptions if possible